backend/src/db/migrations/20230913044708_add_deleted_at_column_to_articles.ts

```diff
@@ -0,0 +1,21 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'articles';
+
+const COLUMN_NAME = {
+  DELETED_AT: 'deleted_at',
+} as const;
```

Comment:
I guess Artem meant that in case if you have one value in enum - then you don't need a enum, but could just have a constant
```suggestion
const DELETED_AT = 'deleted_at',;
```